includes/embeds/class-amp-instagram-embed-handler.php

```diff
@@ -190,6 +117,22 @@ private function create_amp_instagram_and_replace_node( $dom, $node ) {
 		$this->did_convert_elements = true;
```

Comment:
As uncovered in https://github.com/rtCamp/themes/pull/3/files#r423249922, it may make sense in this `create_amp_instagram_and_replace_node()` method to preserve  some inline styles from the `$node` to copy onto `$new_node`.  In particular, `border-radius: 3px; border: 1px solid rgb(219, 219, 219);`. 

Nevertheless, this is is probably not something we should be concerned with. In fact, it's probably actually an AMP issue where `amp-instagram` should be applying those styles. So nevermind!